src/test/java/com/nike/cerberus/service/PermissionsServiceTest.java

```diff
@@ -33,14 +33,15 @@
 import java.util.Optional;
```

Comment:
Looks like your missing a test for the following method:
https://coveralls.io/builds/17944297/source?filename=src%2Fmain%2Fjava%2Fcom%2Fnike%2Fcerberus%2Fservice%2FPermissionsService.java#L200

Reply:
Looks like there was a branch of logic skipped.

Reply:
Given the nature of this service I would like to see 100% branch covereage
- [ ] add more tests